src/components/HdTileSelectItem.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <button
+    class="tile-select-item"
+    type="button"
+    :class="{
+      'tile-select-item--selected': isSelected,
+    }"
+    @click="emitInput"
```

Comment:
We should make the wording more general, as this element is used in different situations. `input` could be enough. Same goes for the next line and line 41.

Reply:
I didn't get it. You suggestion is to rename `emitInput` to `input`?

Reply:
It might be confused, if we rename `emitInput` to `input`.

Reply:
Darn. I read `emailInput`. And I read it multiple times. That's weird. Also took me a short moment to see it now. Looks like I have emit-email weakness. `emitInput` is absolutely fine!

Reply:
Hahahahha 👓 😅